test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +37,40 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementsAreNotNull() throws Exception {
+        // empty list
+        assertIsNoneNull();
+
+        // only one object
+        assertIsNoneNull(1);
+        assertIsNoneNull("a");
+        assertIsNoneNull(new BigInteger("123"));
+        assertIsAnyNull((Object) null);
+
+        // list of one null
+        assertIsAnyNull(1, null, 2);
+        assertIsAnyNull(null, "abc", 2);
+
+        // more than one null
+        assertIsAnyNull(null, null, 2);
+        assertIsAnyNull(2, new Integer(1234), null);
+
+        assertIsNoneNull(1, 2,3 ,4, 5);
+
+
+    }
+
     private void assertAreUnique(Object... objects) {
         assertTrue(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
 
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertIsAnyNull(Object... objects) { assertTrue(Utils.isAnyNull(objects)); }
```

Comment:
Please check the coding standards on how to write methods with just one line of code!